app.js

```diff
@@ -141,6 +142,12 @@
       $th.toggleClass('ascending');
     },
 
+    toggleCorrespondingFilter: function(event) {
+      // Toggles filter when associated label is clicked
+      var $filter = this.$(event.target).siblings().find('.filter');
+      $filter.prop( 'checked', !$filter.prop('checked') );
```

Comment:
They should wrap this in jquery so that you don't have to do this for toggling properties.